src/main/java/pl/piekoszek/gorskimatches/image/LineImageCreator.java

```diff
@@ -2,54 +2,55 @@
 
 import org.springframework.stereotype.Component;
 
-import javax.imageio.ImageIO;
 import java.awt.*;
 import java.awt.image.BufferedImage;
-import java.io.ByteArrayOutputStream;
-import java.io.IOException;
 
 import static java.awt.Color.black;
 
 @Component
 public class LineImageCreator {
 
-    LineInfo[][] newArray = new LineInfo[10][];
+    LineInfo[][] numbersAndSymbols = new LineInfo[62][];
```

Comment:
A bit better, but I'd prefer `lineInfosByCharacter` or `lineInfosByNumberOrSign` or sth like that